### DISCUSSION TITLE
[Review] How I discovered Organic Maps

### DISCUSSION THREAD

**Discussion by jdejaegh** [organicmaps/organicmaps#4514] [Q&A]
Hi there!

I'm not really looking for help, but rather to share how I discovered Organic Maps; as I don't find the Google Play Store very welcoming, I thought I'd share it here.  The description of the repository motivated me to write this review.

> Your donations and positive reviews motivate and inspire our small team! 

Last summer I went to Italy (Piedmont region) with a group of teenagers and was one of the leaders.  I was already familiar with and had contributed to OpenStreetMap and wanted a pocket and offline version of it, just in case.  After a bit of fiddling with F-Droid, I found OsmAnd (which I had used in the past to record gpx tracks) and Organic Maps.  I installed both and downloaded the maps of all the countries we would be crossing.  I was amazed at how simple Organic Maps looked and did exactly what it was supposed to do: let me browse OSM offline and provide routing capabilities.  Much more inviting on first use than OsmAnd.

On our first day in Italy, a bus was supposed to take us up a mountain.  Unfortunately, it broke down halfway up and the driver was unable to take us any closer to our destination.  As quickly as I could hope, I unlocked the phone, launched Organic Maps and found our destination on the map.  We reached it via small hiking trails, which were much more pleasant than the main road; it was nice to see how accurate OSM is and how easy Organic Maps is.

Fast forward to 3pm, after our planned activity.  The bus had to be towed to Milan for repairs, and with public transport on strike and taxis too busy to take people to the airport, our best option was to walk down to Lake Maggiore where a boat could pick us up. A 10km (unplanned) walk in an unfamiliar area after a fairly physical activity; what could go wrong? 

The destination was pinned in Organic Maps, the routing was set to pedestrian to get the shortest route possible, and we were all following the app in a region where mobile coverage is not global.  As much as some of us were annoyed at having to walk, I was enjoying the time, seeing how OSM and Organic Maps helped us find a way to our destination.  Everything I could imagine was there on the map: smaller paths for pedestrians (often shorter than the main roads for cars), fountains to fill our bottles, names of small villages...  All thanks to a little planning before the trip to download the map and (I must admit) a good power bank to keep the phone on!

The app saved us from walking more than necessary because it showed us the shortest route and I could follow it with confidence.  This is how I really discovered Organic Maps.

We also saw some amazing landscapes along the way.  Some pictures are linked below.

<details>
  <summary>On our way following Organic Maps (4 pictures)</summary>

  ![IMG_20220710_193906](https://user-images.githubusercontent.com/31966277/219786310-1d3006c5-1e74-4f3d-a8db-f0a7ab09eeb0.jpg) 

  ![IMG_20220710_194056](https://user-images.githubusercontent.com/31966277/219786402-2f37c1c0-18c2-4c14-a5c1-d85289f3529c.jpg) 

  ![IMG_20220710_200434](https://user-images.githubusercontent.com/31966277/219786433-5d24dfe0-be18-4107-8e6f-f7db4e7e1c23.jpg) 

  ![IMG_20220710_212139](https://user-images.githubusercontent.com/31966277/219786443-5a2fcda5-d3c3-4911-82d5-89d60cc93ea3.jpg) 

</details>

Sure, the app could have more features, and the 1.4k open issues are certainly relevant.  But it does what it is supposed to do and does it well.  And most importantly, it does it when you need it! 

I am going back to Italy soon, again with a group of teenagers, and guess what?  Organic Maps will be with me too!

A big thank you to the people making this happen!

How did you find out about Organic Maps?  Looking forward to hearing about it!

**Comment by ypyl** [comment]
Using it for navigation every day after 'maps me' has been bought. Never had any issue. Not sure about 1,4 opened issues and I am really surprised that there are issues.
Great and mature product. Thank you very much for your work and passion!
Really like how fast maps are updated from OpenStreetMap. As it allows me to edit maps myself and see changes next update.

**Comment by ghost** [comment]
Amazing experience as people above, including big trip to the Netherlands and England just last month !
Also used daily to look up countries/places/areas when I hear about them in the media, building up my bookmarks !
